tasks/packages.yml

```diff
@@ -17,8 +17,30 @@
 # limitations under the License.
 #
 
+- include: packages_ubuntu.yml
+  when: ansible_distribution == 'Ubuntu'
```

Comment:
I thought `ansible_distribution` is more appropriate than `ansible_os_family` because this role will only work on Ubuntu and not Debian (I think), opposed to `ansible_os_family == 'RedHat'` which will cover all the variants of RedHat including CentOS and Scientific Linux and such which all can use CentOS's OpenStack repository.

Though, now that I think about it, maybe we can use `ansible_pkg_mgr` will be more appropriate if we focus on packaging system, which will be `apt` on Ubuntu and `yum` on EL. What do you think (though, we still need to rely on `ansible_distribution` or `ansible_os_family` to assign package name variable)?